src/main/java/oss/fosslight/service/impl/OssServiceImpl.java

```diff
@@ -1586,6 +1593,7 @@ public void checkOssLicenseAndObligation(OssMaster ossMaster) {
 		
 		for(OssLicense license : ossMaster.getOssLicenses()) {
 			LicenseMaster master = CoCodeManager.LICENSE_INFO_UPPER.get(license.getLicenseName().toUpperCase());
+			master = master == null ? new LicenseMaster() : master;

```

Comment:
just creating it can cause Null Point Exception when using getters.

ex)
1595 : LicenseMaster master = null 
1596 : master = new LicenseMaster();
master.get~ (Null Pointer Exception)